app/models/service_provider.rb

```diff
@@ -107,7 +107,7 @@ def logo_file_mime_type
     return unless logo_file.attached?
     return if mime_type_valid?
     errors.add(:logo_file, "The file you uploaded (#{logo_file.filename}) is not a PNG or SVG")
-    logo_file.purge
+    logo_file = nil # rubocop:disable Lint/UselessAssignment
```

Comment:
can we remove the line instead of doing the assignment? I'm not super familiar with how the logo files get stored, does it have something to do with that?

Reply:
Yes: https://github.com/rails/rails/issues/37069#issuecomment-532462387

Reply:
👍🏼